app/src/main/res/layout/dialog_info_map_mode.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
It could be easier to position all elements using constraint layout

Reply:
Even wrap_content size did not change that?

Reply:
No it didn't work. 
https://stackoverflow.com/questions/28513616/android-get-full-width-for-custom-dialog
Here, the most ideal solution was using relative layout, in order to avoid style changes etc.